### PR TITLE
Doc: Update image rendering configuration

### PR DESCRIPTION
**Pull request by AgnesToulet**
**What is this feature?**
Update image rendering documentation around configuration.

**Why do we need this feature?**
This PR (https://github.com/grafana/grafana-image-renderer/pull/477) adds new configuration option to the image renderer and the doc should be updated accordingly.

**Who is this feature for?**
Image renderer users

### CODE REVIEW

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 1]
```diff
@@ -2265,6 +2265,14 @@ Instruct headless browser instance whether to output its debug and error message
 
 It can be useful to set this to `true` when troubleshooting.
 
+### rendering_timing_metrics
+
+> **Note:** Available from grafana-image-renderer v3.9.0+
+
+Instruct headless browser instance whether to register metrics for the duration of every rendering step. Default is `false`.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-grafana/_index.md:2272]:
```suggestion
Instruct a headless browser instance on whether to record metrics for the duration of every rendering step. Default is `false`.
```

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 2]
```diff
@@ -2265,6 +2265,14 @@ Instruct headless browser instance whether to output its debug and error message
...
+
+It can be useful to set this to `true` when optimizing the rendering mode settings to improve the plugin performance or when troubleshooting.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-grafana/_index.md:2274]:
```suggestion
Setting this to `true` when optimizing the rendering mode settings to improve the plugin performance or when troubleshooting can be helpful.
```

File: docs/sources/setup-grafana/image-rendering/_index.md [hunk 3]
```diff
@@ -229,6 +229,25 @@ ENABLE_METRICS=true
 }
 ```
 
+#### Enable detailed timing metrics
+
+With the Prometheus metrics enabled (see previous paragraph), you can also enable detailed metrics to get the duration of every rendering step.
```
Comment (chri2547) [inline docs/sources/setup-grafana/image-rendering/_index.md:234]:
```suggestion
With the [Prometheus metrics enabled](#enable-prometheus-metrics), you can also enable detailed metrics to get the duration of every rendering step.
```

**Review by chri2547 — COMMENTED** [review]
Added a few doc suggestions. Thank you for your contribution!

**Comment by AgnesToulet** [comment]
Thanks, I updated the PR with your suggestions 👍

**Review by chri2547 — COMMENTED** [review]
Docs approved. Awaiting engr review on this PR.

**Review by evictorero — APPROVED** [review]
LGTM

**Pull request merged**